java-squid/src/main/java/org/sonar/java/Measurer.java

```diff
@@ -99,7 +99,7 @@ public void scanFile(JavaFileScannerContext context) {
     context.addNoSonarLines(commentLinesVisitor.noSonarLines());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Method 'JavaFileScannerContext.addNoSonarLines(...)' is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)